.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Shouldn't it be the "ENHANCEMENT" label? 🤔

Reply:
Updated in https://github.com/nomtek/mistralai_client_dart/pull/25/commits/8cd768d5fd5da1da3dfa28e4f126b264a99808cb